content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -7,39 +7,59 @@ preview: true
 
 # Full-Text Search (Preview)
 
-The Prisma Client supports a full-text search API for Postgres databases. With full-text search enabled, you can add search functionality to your application by searching for text within a database column.
+The Prisma Client supports full-text search API for **PostgreSQL** and **MySQL** databases. With full-text search enabled, you can add search functionality to your application by searching for text within a database column.
 
-<Admonition type="info">
+## Enabling full-text search
 
-**Note**: Full text search can **only** be used in a Postgres database
-
-</Admonition>
-
-## Enable full-text search
-
-The full-text Search API is still in **Preview**. You can enable `fullTextSearch` in your Prisma Schema:
+The `fullTextSearch` feature is currently a Preview feature. To enable this feature, carry out the following steps:
```

Comment:
`fullTextSearch` is not the feature, but the preview feature flag. The feature is indeed the full-text Search API as it was spelled out in the past.

Reply:
Thanks, fixed